azure/iac/Redis.rego

```diff
@@ -0,0 +1,95 @@
+package rule
+
+# https://docs.microsoft.com/en-us/azure/templates/Microsoft.Cache/redis
+
+# PR-AZR-0131-ARM
+
+default serverRole = null
+
+azure_attribute_absence ["serverRole"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.cache/redis"
+    not resource.properties.enableNonSslPort
+}
+
+azure_issue ["serverRole"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.cache/redis"
+    resource.properties.enableNonSslPort != false
+}
+
+serverRole {
+    azure_attribute_absence["serverRole"]
+}
+
+
+serverRole {
+    lower(input.resources[_].type) == "microsoft.cache/redis"
+    not azure_issue["serverRole"]
+}
+
+
+serverRole = false {
+    azure_issue["serverRole"]
+}
+
+serverRole_err = "Azure Redis Cache enableNonSslPort is not false currently on the resource" {
```

Comment:
this message need to be more generic than technical when value does not match. see other rules for example.

Reply:
I put this error message according the other rules
Can you please show me an example for this?

Reply:
@shahinshirvani  Well according to the policy title the message can be` "Redis cache is currently allowing non ssl connection."` or `"Redis cache is currently allowing unsecure connection via a non ssl port opened"`
Which ever you think best can use.

We have remediation description where we can elaborate about `enableNonSslPort `property.

Reply:
fixed